lib/mini_defender/data/public_domain_patterns.txt

```diff
@@ -0,0 +1,130 @@
+# Generic TLDs
```

Comment:
This list does not cover all valid public suffixes, please use the `ICANN DOMAINS` list from: https://publicsuffix.org/list/public_suffix_list.dat

Reply:
[Here](https://github.com/weppos/publicsuffix-ruby/tree/main) is the `public_suffix` gem; that supports Mozilla initiative